src/Compilers/CSharp/Test/Syntax/Parsing/RawInterpolatedStringLiteralParsingTests.cs

```diff
@@ -0,0 +1,1420 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing;
+
+public class RawInterpolatedStringLiteralParsingTests : CSharpTestBase
+{
+    #region Single Line
+
+    [Fact]
+    public void SingleLine1()
+    {
+        var text = @"
+class C
```

Comment:
For all these tests, you could use a top-level statement and verify execution (with `System.Console.Write($""""" """""");`) instead.
`CreateCompilation` infers the `exe` emit option when a top-level statement is present.

Reply:
i believe that's what @jnm2 's tests do :)

Reply:
I see. Looks like the tests that execute are elsewhere, so these purely focus on parsing. Still, compacting these wouldn't hurt.

Reply:
Having looked at other tests, I couldn't find where those tests are. Is it a different PR?

Reply:
Gentle ping

Reply:
Sure.  those tests are in C:\github\roslyn\src\Compilers\CSharp\Test\Syntax\Parsing\RawInterpolatedStringLiteralCompilingTests.cs

Any test that passed `expectedOutput: "..."` is validating in this fashion.